build_functions/CMakeLists.txt

```diff
@@ -193,3 +193,22 @@ function(compileTargetAsC11 theTarget)
     set_target_properties(${theTarget} PROPERTIES CXX_STANDARD 11)
   endif()
 endfunction()
+
+set(LIBS_TO_BE_REAL_CHECKED CACHE INTERNAL "LIBS_TO_BE_CHECKED")
+
+function (add_library())
```

Comment:
>add_library [](http://example.com/codeflow?start=10&length=11)

Challenge: can you test this by checking that the libs we expect are there in c-build-tools?

As in if the LIBS_TO_BE_REAL_CHECKED does not contains what we expect for c-build-tools, then let's have an error in cmake.

Reply:
Nice, I saw the test